backend/municipal-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -0,0 +1,147 @@
+        raise HTTPException(status_code=500, detail=f"Failed to process vendor data: {str(e)}")
+
+    finally:
+        if input_temp_file and os.path.exists(input_temp_file.name):
+            os.unlink(input_temp_file.name)
+
+
+@router.get('/getFacilityIngestionTemplate',
+            summary='Generate facility ingestion template Excel file with schema and boundary codes',
+            response_description="Returns Excel template with facility schema and boundary codes")
+async def get_facility_ingestion_template(
+        facility_service: FacilityTemplateService = Depends(),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)